site/docs/v1/tech/guides/account-management.adoc

```diff
@@ -0,0 +1,190 @@
+---
+layout: doc
+title: Account Management
+description: Managing Accounts
+---
+:page-liquid:
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+[NOTE.since]
+====
+Available since 1.26.0
+====
+
+== Overview
+
+This allows you the manage accounts.
```

Comment:
Can we blow out this statement a bit further? 

Something like: 

> FusionAuth self-service account management allows your users to manage their own accounts.

Reply:
Makes sense